lib/proc/proc_select.cpp

```diff
@@ -3745,6 +3745,7 @@ grn_select_apply_adjuster_execute_adjust(grn_ctx *ctx,
     options.similarity_threshold = 0;
     options.max_interval = 0;
     options.additional_last_interval = 0;
+    options.min_interval = 0;
```

Comment:
OK, I will  change to use `DEFAULT_MIN_INTERVAL`.
(I think `options.max_intervals` should also use `DEFAULT_MAX_INTERVAL`, through...)

Reply:
`DEFAULT_MIN_INTERVAL` is defined at `expr.c`.
So in order to use it in `proc_select.cpp`, we should move it and same definition like `DEFAULT_MAX_INTERVAL` to `expr.h` and rename them like `GRN_EXPR_DEFAULT_MAX_INTERVAL` because `DEFAULT_MAX_INTERVAL` is common name and will be duplicated.

As a conclusion, how about do that as a follow up task in another pull request?

Reply:
This line is modified to `options.min_interval = NULL;` because min_interval is changed to `int *` type.

https://github.com/groonga/groonga/pull/1520#discussion_r1116652096

Also, I have moved `DEFAULT_MAX_INTERVAL` into `grn_ii.h` and renamed it as `GRN_II_DEFAULT_NEAR_MAX_INTERVAL` as the comment above.
I think it is better to move other related definitions into `grn_ii.h`, but it should be worked in another PR.